messaginginapp/src/main/java/io/customer/messaginginapp/store/InAppPreferenceStore.kt

```diff
@@ -9,6 +9,28 @@ interface InAppPreferenceStore {
     fun saveNetworkResponse(url: String, response: String)
     fun getNetworkResponse(url: String): String?
     fun clearAll()
+
+    // Broadcast message storage with expiry
+    fun saveBroadcastMessages(messages: String, expiryTimeMillis: Long)
```

Comment:
This looks a long list of methods. I think using an enum/data class with fewer methods to manage states might make it easier to read and maintain.